.github/workflows/install-test.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  schedule:
+    - cron: '0 0 * * *' # Every day at midnight
```

Comment:
why only at midnight? it seems like there's probably value in running this more frequently

Reply:
It seems like _maybe_ artifactory might rate limit requests ([one of these runs resulted in a 403](https://github.com/stripe/stripe-cli/actions/runs/8790803561/job/24123652260)), so that would be something to be aware of. But we could easily do this once an hour if we want